From a056f22eae7c48a6f59307e35694d090c0abf975 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 07 Oct 2018 10:51:37 +0300
Subject: [PATCH] Makefile: release: add more repositories to the list

Add dependency repositories to the RELEASE_REPOS list.
Also, while at this sort the list.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index 8b5921a..92c4e37 100644
--- a/Makefile
+++ b/Makefile
@@ -67,7 +67,8 @@
 	$(error EMBD_RELEASE_VER is undefined, usually looks like x.y.z)
 endif
 
-RELEASE_REPOS := buildroot-external preboot u-boot linux armcp tbot fw_loader btl
+RELEASE_REPOS := 3rd-parties armcp btl buildroot buildroot-external \
+	fw_loader linux preboot specs specs_external tbot u-boot
 RELEASE_TAG_POINTER := latest-embedded-release
 RELEASE_BRANCH := $(shell \
 	git branch -r --points-at HEAD | \
